avail-subxt/examples/da_bridge_actor.rs

```diff
@@ -0,0 +1,57 @@
+use std::{thread, time};
+
+use avail_subxt::*;
+use sp_keyring::AccountKeyring;
+use subxt::{ext::sp_core::H160, tx::PairSigner, OnlineClient};
+
+const DESTINATION_DOMAIN: u32 = 1000;
+const DA_BRIDGE_ROUTER_ADDRESS: &str = "0x77534486c6467fd24b1f7d60ca61d984d91f6a2a";
+
+#[async_std::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
Should we remove unwraps in while loop code to avoid confusion when someone blindly copy example? I've tried to improve it, not sure if better represents flow or not.
```
async fn process_header(
	api: &OnlineClient<AvailConfig>,
	header: DaHeader,
	signer: &PairSigner<AvailConfig, Pair>,
) -> Result<()> {
	println!("Finalized block header: {:?}", &header);
	println!("Header data root: {:?}", header.extrinsics_root.data_root);

	let bridge_router_eth_addr: H160 = DA_BRIDGE_ROUTER_ADDRESS.parse()?;
	let tx = avail::tx().da_bridge().try_enqueue_data_root(
		DESTINATION_DOMAIN,
		bridge_router_eth_addr.into(),
		header.into(),
	);

	println!(
		"Sending finalized block header. Domain: {}. Recipient: {}",
		DESTINATION_DOMAIN, DA_BRIDGE_ROUTER_ADDRESS
	);
	let tx_progress = api
		.tx()
		.sign_and_submit_then_watch(&tx, signer, Default::default())
		.await?;

	let h = tx_progress.wait_for_finalized_success().await?;
	let submitted_block = api
		.rpc()
		.block(Some(h.block_hash()))
		.await?
		.ok_or_else(|| anyhow!("no block"))?;

	let xts = submitted_block.block.extrinsics;
	println!("Submitted block extrinsic: {xts:?}");
	Ok(())
}

#[async_std::main]
async fn main() -> Result<(), Box<dyn std::error::Error>> {
	let api = OnlineClient::<AvailConfig>::from_url("ws://127.0.0.1:9944").await?;
	let signer = PairSigner::new(AccountKeyring::Alice.pair());

	let mut finalized_blocks = api.rpc().subscribe_finalized_blocks().await?;

	while let Some(finalized_block) = finalized_blocks.next().await {
		match finalized_block {
			Ok(header) => {
				if let Err(error) = process_header(&api, header, &signer).await {
					println!("Error: {}", error);
				}
			},
			Err(error) => println!("Error: {}", error),
		}

		println!("Sleeping for 10 sec...\n");
		thread::sleep(time::Duration::from_secs(10));
	}

	Ok(())
}
```